matRad_calcDoseDirectMC.m

```diff
@@ -1,4 +1,4 @@
-function resultGUI = matRad_calcDoseDirectMC(ct,stf,pln,cst,w,nHistories)
+function resultGUI = matRad_calcDoseDirectMC(ct,stf,pln,cst,w,nHistories,exportForExternalCalculation)
```

Comment:
The parameter has been moved to the TOPASconfig and documentation has been added

Reply:
By the way - I would make this a pln parameter. It's better for keeping interface compatibility.

Reply:
With the config updates, it will show up in pln.propMC.externalCalculation. But the reason it is part of the topasConfig is that it's only needed for TOPAS, so in my view, it makes sense to be in the specifig config.